client/src/app/+admin/users/user-list/user-list.component.ts

```diff
@@ -92,12 +103,47 @@ export class UserListComponent extends RestTable implements OnInit {
         }
       ]
     ]
+
+    this.columns = [
+      { key: 'username', label: 'Username' },
+      { key: 'email', label: 'Email' },
+      { key: 'quota', label: 'Video quota' },
+      { key: 'role', label: 'Role' },
+      { key: 'createdAt', label: 'Created' }
+    ]
+    this.selectedColumns = [...this.columns]
```

Comment:
`[ ...this.columns ]` + please add a small comment to explain the purpose of this line (make a copy of the array)